### PR TITLE
hfflavour2_changeModels

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
### **bert-base-cased:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/e4d06c56-aaba-4c4b-a951-f625cd3cca59/runs/maroon_pizza_tgq5np6nh9?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point but find the screen shot for the same

image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/01da5401-bfa9-4f67-90f3-6d08a6447324/runs/batchjob-ca069b5e-25ea-4395-bfe5-894cd14010b3?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/bert-base-cased:2/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`

**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForMaskedLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: bert-base-cased
    model_data: data
    pytorch_version: 1.13.1
    task_type: fill-mask
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: 813712038bca4de191f834e09ca2bdb9
utc_time_created: '2023-10-31 11:11:58.639094'
`
### **bert-base-uncased:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/6013b75c-259b-4d50-bf16-6101e373a091/runs/be9735e3-fbd8-47f7-8b60-43f9c2d94f5b?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point but find the screen shot for the same
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/79ebef24-f838-4fb3-afc6-c4079f5f485e/runs/batchjob-fe90e8b6-bc87-4a05-b049-ac2d1bf0b90f?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/bert-base-uncased:3/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`

**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForMaskedLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: bert-base-uncased
    model_data: data
    pytorch_version: 1.13.1
    task_type: fill-mask
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: bda55a837ef749d5935229b197bcd0b1
utc_time_created: '2023-10-31 16:30:32.178295'
`

### **gpt2:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/7fbc6b16-5a12-4724-bcb5-936e5db4046a/runs/lemon_box_fvzqzctccq?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fv-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point but find the screen shot for the same
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/3e2c1643-48bd-4c30-886e-31c4ac3ad9c4/runs/batchjob-140acba7-f9a0-48d2-8a5f-4320e79c0d00?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/gpt2:5/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`
**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelWithLMHead
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: gpt2
    model_data: data
    pytorch_version: 1.13.1
    task_type: text-generation
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: ccc10b1497944cc2bd5235c4ee00e36b
utc_time_created: '2023-10-31 11:14:21.742963'
`

### **distilgpt2:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/3494c3fd-5516-479e-b2c4-368cab472076/runs/neat_thread_w0s18jzbmd?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point but find the screen shot for the same
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/05020c19-e50e-4a97-938b-0c0a82496695/runs/batchjob-f1c04a66-fc41-4bb1-9415-2255033757fc?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/distilgpt2:3/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`

**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelWithLMHead
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: distilgpt2
    model_data: data
    pytorch_version: 1.13.1
    task_type: text-generation
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: 131bee2d935d46f5aaf92369121cc258
utc_time_created: '2023-10-31 11:08:58.100785'
`

### **deepset-minilm-uncased-squad2:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/0511c399-a19d-44f3-89db-935ed4f745dd/runs/gray_carrot_2tp0khwrtg?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point but find the screen shot for the same
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/9f6edc7e-21bd-4b5b-9832-c303b7b5009b/runs/batchjob-ae4a9353-1e72-4750-bd72-850546b0d1be?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/deepset-minilm-uncased-squad2:4/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`
**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForQuestionAnswering
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: deepset/minilm-uncased-squad2
    model_data: data
    pytorch_version: 1.13.1
    task_type: question-answering
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: f38f32e0a06a42cd92a7969c2db9ecf0
utc_time_created: '2023-10-31 12:14:03.113058'

### **microsoft-deberta-base:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/a4489a37-9a29-4629-bafe-9d12a374f63a/runs/19310508-fd37-4043-b901-a8e07824997a?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fv-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point added the screen shot for the same

The issue is with the model in HF checked inference API compared the files found no issue with files
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/2553dcde-c392-467a-8e91-32b7a1ef9223/runs/batchjob-cba35453-019d-4436-91d1-5b83e472e9ce?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/microsoft-deberta-base:1/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`

**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForMaskedLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: microsoft/deberta-base
    model_data: data
    pytorch_version: 1.13.1
    task_type: fill-mask
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: d774d3cab7e64c9a8036b5f1350572ea
utc_time_created: '2023-10-31 17:26:00.525461'

### **microsoft-deberta-large:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/2476c2b6-fac4-48a2-97ca-7085fc2784e0/runs/gray_stem_tmrvf8rxpx?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point added the screen shot for the same

The issue is with the model in HF checked inference API compared the files found no issue with files
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/experiments/id/207057eb-5abd-43f9-b083-7ed10f276fc2/runs/batchjob-7b545d4b-2112-425b-812a-11d52edc4ec5?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/microsoft-deberta-large:3/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`
**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForMaskedLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: microsoft/deberta-large
    model_data: data
    pytorch_version: 1.13.1
    task_type: fill-mask
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: 567842e9caf64b70b7147c354ad6826c
utc_time_created: '2023-11-01 16:53:38.100220'
`
-----------------------

### **roberta-base-openai-detectorroberta-base-openai-detector:-**

**Run link with all expected validations passing**

[Import Model Validation](https://ml.azure.com/experiments/id/5fb649c8-8f56-47f4-bdac-72b62a5538ac/runs/polite_jewel_xfm0td7v2c?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint link and test Inference snapshots**

Due to quota issue we delete end point added the screen shot for the same
image

**Batch inference successful run link**

[Batch inference link](https://ml.azure.com/endpoints/realtime/robertabaseopenaidetector1/detail?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Model conda.yaml  and model files link:**
[Files Link](https://ml.azure.com/model/roberta-base-openai-detector:3/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Conda yaml code**
`channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env
`
**MLmodel file  code**
`flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForSequenceClassification
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: roberta-base-openai-detector
    model_data: data
    pytorch_version: 1.13.1
    task_type: text-classification
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: 1b3b17d6f3e54679b9ddbeac468a3a1a
utc_time_created: '2023-11-02 13:59:16.064671'
`

### CODE REVIEW

File: assets/models/system/distilgpt2/spec.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ properties:
     Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2,
     Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E2s_v3, Standard_E4s_v3,
     Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3,
-    Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3,
+    Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3
     Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4,
```
Comment (BhavanaTumma) [inline assets/models/system/distilgpt2/spec.yaml:22]: comma to be added.

File: assets/models/system/deepset-minilm-uncased-squad2/description.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Question Answering|Extractive Q&A|<a href="https://huggingface.co/datasets/squad
 #### Sample input
 ```json
 {
-    "inputs": {
+    "input_data": {
         "question": ["What is my name?", "Where do I live?"],
         "context": ["My name is John and I live in Seattle.", "My name is Ravi and I live in Hyderabad."]
     }
```
Comment (BhavanaTumma) [inline assets/models/system/deepset-minilm-uncased-squad2/description.md:37]: This example seems confusing, could we use a better one?

Comment (KonakanchiSwathi) [reply assets/models/system/deepset-minilm-uncased-squad2/description.md:37]: @skasturi @novaturient95  can you please comment is this wrong or do we need to change the input data

**Comment by skasturi** [comment]
Checked the models registered to the registry. They seem to be missing License files?

**Review by skasturi — APPROVED** [review]

**Pull request merged**